.angular-cli.json

```diff
@@ -22,7 +22,8 @@
         "../node_modules/pikaday/css/pikaday.css",
         "../node_modules/pikaday/css/triangle.css",
         "../node_modules/c3/c3.css",
-        "styles.css"
+        "styles.css",
+        "../node_modules/ngx-prx-styleguide/styles.css"
```

Comment:
normally I would want to use the styleguide before the app styles.css, but reset is doing all kinds of things to us that need to be cleaned up before we can override styleguide styles with app specifics